### PR TITLE
#190 convert team page to JSON

### PR DESCRIPTION
**Pull request by xming13**
Converted to JSON!

### CODE REVIEW

File: app/scripts/about/templates/team.hbs [hunk 1]
```diff
@@ -2,93 +2,39 @@
   <div class="col-md-6 col-md-offset-3" id="team">
     <h1 class="nm-page-heading">The Team</h1>
     <hr>
-    <p>NUSMods is an open source project that relies on the continuous support of its individual contributors and NUS student community. We hope to involve more developers and contributors in making NUSMods even better. Please reach out to us if you are interested in helping!
+    <p>NUSMods is an open source project that relies on the continuous support of its individual contributors and NUS
+      student community. We hope to involve more developers and contributors in making NUSMods even better. Please reach
+      out to us if you are interested in helping!
+
```
Comment (xinan) [inline app/scripts/about/templates/team.hbs]: Oops! I didn't notice that the closing `</p>` was missing. Please add it in also. Thanks!

File: app/scripts/about/templates/team.hbs [hunk 2]
```diff
@@ -2,93 +2,39 @@
...
+                <a href="{{linkedin}}" target="_blank"><i class="fa fa-linkedin-square fa-lg"></i></a>
+              {{/if}}
+            </p>
+          </div>
+        </div>
+      {{/each}}
+    {{/if}}
   </div>
-</div>
+</div>
```
Comment (xinan) [inline app/scripts/about/templates/team.hbs]: Leaving a newline at the end is good practice! :smile: Please add it in also. Thanks!

**Comment by xming13** [comment]
Fixed. :)

**Comment by xinan** [comment]
Not sure if it is better to just use `name` as the `alt` for the ``. Leave the rest to @yangshun :stuck_out_tongue:

**Comment by yangshun** [comment]
It's fine, thanks Xiao Ming! Btw, Xiao Ming (Wei Seng), is my JC classmate who has already graduated from NUS. Thanks for helping :D

Closes #190

**Comment by xming13** [comment]
Glad to help! :smile:

**Pull request merged**